### PR TITLE
Changed user/activation limit validation to CIDR

### PR DESCRIPTION
**Pull request by sasbury**
src network is now a comma separated list of CIDR
This matches NSC

Activation and user use the same limit object. Updated that object and the tests.

### CODE REVIEW

File: types.go [hunk 1]
```diff
@@ -185,10 +186,13 @@ func (l *Limits) Validate(vr *ValidationResults) {
 	}
 
 	if l.Src != "" {
-		ip := net.ParseIP(l.Src)
+		elements := strings.Split(l.Src, ",")
```
Comment (derekcollison) [inline types.go]: Do we know if parser deals with spaces or should we trim the elements array here?

File: user_claims_test.go [hunk 2]
```diff
@@ -314,3 +314,56 @@ func TestUserAccountIDValidation(t *testing.T) {
...
+
+	uc.Limits.Src = "192.0.2.1/1"
+	vr = CreateValidationResults()
+	uc.Validate(vr)
+
+	if !vr.IsEmpty() {
+		t.Error("limits should be valid")
+	}
+
+	uc.Limits.Src = "192.0.2.0/24,2001:db8:a0b:12f0::1/32"
```
Comment (derekcollison) [inline user_claims_test.go]: Maybe add in a test with different spaces or tabs around the comma separations.

Comment (sasbury) [reply user_claims_test.go]: Added support for whitespace and updated test in user limits

File: types.go (hunk 1 above)
Comment (sasbury) [reply types.go]: didn't but added that

**Pull request merged**